Resources/config/services.xml

```diff
@@ -66,21 +58,21 @@
 
         <!-- Manager -->
         <service id="sulu_form.manager.form"
-                 class="Sulu\Bundle\FormBundle\Manager\FormManager">
+                 class="Sulu\Bundle\FormBundle\Manager\FormManager" public="true">
             <argument type="service" id="doctrine.orm.entity_manager" />
             <argument type="service" id="sulu_form.repository.form" />
             <argument type="service" id="`" on-invalid="null" />
         </service>
 
         <!-- Content Types -->
-        <service id="sulu_form.content_type.form_select"
-                 class="Sulu\Bundle\FormBundle\Content\Types\FormSelect">
+        <service id="sulu_form.content_type.single_form_selection"
+                 class="Sulu\Bundle\FormBundle\Content\Types\SingleFormSelection">
             <argument>SuluFormBundle:Template:content-types/form_select_simple.html.twig</argument>
             <argument type="service" id="sulu_form.repository.form" />
             <argument type="service" id="sulu_form.builder" />
             <argument type="service" id="sulu_form.reference_store.form"/>
 
-            <tag name="sulu.content.type" alias="form_select"/>
+            <tag name="sulu.content.type" alias="single_form_selection"/>
```

Comment:
Add the following to the UPGRADE.md:

~~~~
# dev-develop

The content type for form selection has been changed from `form_select` to `single_form_selection` also the param `type` has changed to `resourceKey`:

**Before**

```xml
<property name="form" type="form_select">
    <params>
        <param name="type" value="page" />
    </params>
</property>
```

**Before**

```xml
<property name="form" type="single_form_selection">
    <params>
        <param name="resourceKey" value="page" />
    </params>
</property>
```
~~~~